sonic-net/sonic-buildimage
# support asic vendor clounix series products

#### Why I did it

To meet market trend, CLOUNIX has provided networking solution and enjoy sharing with the partners. 

#### How I did it
Implement the drivers for the CLOUNIX asic and it works well.

#### How to verify it
generate SONiC image by "make target/sonic-clounix.bin" and verify it on the CLOUNIX platform.


#### Which release branch to backport (provide reason below if selected)




#### Tested branch (Please provide the tested image version)




#### Description for the changelog




#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
